### PR TITLE
Remove ambiguous symbols - TCHAR & generic_string

### PR DESCRIPTION
**Pull request by donho**
The "ambiguous symbols (TCHAR & generic_string) are used when Notepad++ suppoted both ANSI & Unicode formats. Since ANSI format of Notepad++ has been dropped (since long time ago), TCHAR & generic_string will be replaced by wchar_t & wstring respectively.

### CODE REVIEW

File: PowerEditor/src/MISC/Common/Common.cpp [hunk 1]
```diff
@@ -31,13 +31,13 @@
 
 void printInt(int int2print)
 {
-	TCHAR str[32];
+	wchar_t str[32];
 	wsprintf(str, TEXT("%d"), int2print);
 	::MessageBox(NULL, str, TEXT(""), MB_OK);
```
Comment (xomx) [inline PowerEditor/src/MISC/Common/Common.cpp:38]: now it should be:
```
	wsprintfW(str, L"%d", int2print);
	::MessageBoxW(NULL, str, L"", MB_OK);
```

File: PowerEditor/src/MISC/Common/Common.cpp [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
+	wchar_t str[32];
 	wsprintf(str, TEXT("%d"), int2print);
 	::MessageBox(NULL, str, TEXT(""), MB_OK);
 }
 
 
-void printStr(const TCHAR *str2print)
+void printStr(const wchar_t *str2print)
 {
 	::MessageBox(NULL, str2print, TEXT(""), MB_OK);
```
Comment (xomx) [inline PowerEditor/src/MISC/Common/Common.cpp:44]: ::MessageBoxW(NULL, str2print, L"", MB_OK);

File: PowerEditor/src/MISC/Common/Common.cpp [hunk 3]
```diff
@@ -50,7 +50,7 @@ generic_string commafyInt(size_t n)
 	return ss.str();
 }
 
-std::string getFileContent(const TCHAR *file2read)
+std::string getFileContent(const wchar_t *file2read)
 {
 	if (!::PathFileExists(file2read))
```
Comment (xomx) [inline PowerEditor/src/MISC/Common/Common.cpp:57]: PathFileExistsW

@donho 
I will not continue in this review, there are so many problems of this type.
E.g. the nearby line 63 should be: `FILE *fp = _wfopen(file2read, L"rb");`

**Review by xomx — COMMENTED** [review]
@donho 
This PR is not completed.
- the `TEXT("")` and `_T('')` should be replaced by the `L""`.
- A- and T-WINAPIs replaced by the W-variants (`MessageBoxA` and `MessageBox` -> `MessageBoxW`) where appropriate
- ditto A-func and T-funcs with W-funcs ( `fopen` and `_tfopen` -> `_wfopen` )

File: PowerEditor/src/MISC/Common/Common.cpp (hunk 1 above)
Comment (donho) [reply PowerEditor/src/MISC/Common/Common.cpp:38]: @xomx 
Agree for the TEXT("") and _T('') being replaced by the L"", to make the intention (semantic) of using char or wchar_t more clear. However, adding suffix "W" for all the win32 functions is not necessary to me.

**Pull request closed without merging**